lib/engines/content_object_store/app/lib/content_object_store/publishable.rb

```diff
@@ -51,6 +52,15 @@ def update_content_block_document(new_content_block_edition:, update_document_pa
 
   private
 
+    def remove_cache_for_dependent_content(content_block_edition:)
```

Comment:
just realised I'm switching between dependent and host content... i've seen both used elsewhere, shall we stick with Host in the code as that's what we've got already?

Reply:
cool, have changed to use host